src/config/webpack.config.js

```diff
@@ -3,6 +3,8 @@
 const path = require('path')
 const webpack = require('webpack')
 const merge = require('webpack-merge')
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
```

Comment:
There shouldn't be semi-colons at the end of the line according to StandardJS. (same on the line below).
Also two lines below doesn't put spaces after/before the curly brackets. Though I don't know which style we are actually using.